bigbang/mailman.py

```diff
@@ -24,7 +24,8 @@
 
 from config.config import CONFIG
 
-from . import listserv, parse, w3crawl
+from bigbang.ingress import listserv, w3c
+from bigbang import parse
```

Comment:
These changes are going to create a merge conflict with #540

Reply:
I will sort them out once you merged that PR into main